src/main/java/com/yolo/customer/AI/PromptBuilder.java

```diff
@@ -14,9 +18,9 @@ public static String buildPrompt(AIRequest requestBody) {
         String restrictionsString = String.join(", ", restrictions);
 
         // Building the custom prompt with interests and restrictions
-        return "Generate a unique idea for the chef that includes a title and description. The title should be no more than 64 characters, and the description should be no more than 128 characters. "
+        return "Generate a unique idea for the chef that includes a title and description. The title should be no more than 64 characters, and the description should be no more than" + maxLimit + "characters. "
```

Comment:
move the prompt to a separate file in resources folder instead of hardcoding in code